templates/apps/create.html

```diff
@@ -195,7 +195,7 @@ <h4>Important information</h4>
 
```

Comment:
Just a general statement. This create form is sooo hard to understand.  We have to simplify this!

Reply:
Agreed!